app/src/app/list/list.component.sass

```diff
@@ -1,3 +1,7 @@
+@import '../../responsive'
+
 .poll-list
   margin-top: 3em
-  
\ No newline at end of file
+
+#tab-polls
+  margin-top: -40px
```

Comment:
This margin is wrong, mainly because you're cancelling a margin that is expressed in em unit. If the font size changes, the margin of .main-container will change, but not this margin.

In fact, to do things right here, you'll have to fix the margins of .main-container, which are wrong. The top margin of .main-container has two purposes: compensate the navtitle bar, and and some space below the bar. The height of navtitle is store in $navbarheight in sizes.sass, and it is expressed in pixels. The space below the bar must depend on the font size and must be expressed in ex (not em which are horizontal spaces). Of course, it is not possible to add a value in px to a value in ex. Therefore, .main-container must have both a top margin (of the same height as navtitle) and a top padding (expressed in ex). The value of the top padding must be stored in a variable in sizes.sass. You'll use this variable in your negative margin (something like `-1 * ($var - 2ex)`).

Reply:
I was wrong!